tests/bins/elf/symver/Makefile

```diff
@@ -0,0 +1,22 @@
+# Build shared libraries (32/64) with versioned symbols and applications (32/64).
```

Comment:
this is unrelated to your patch but I realized goblin is using an `include` section, and we include `tests` wholesale; since I believe `include` is only for crates downloads, we should only include what this crate needs to build it's source as a dependency, not it's tests; so all this stuff is adding a lot of unnecessary downloads to people.

Reply:
Honesty I don't like to commit binaries, but I didn't see a good way at the time, and I saw other binaries. I compiled them with `-Os` but it doesn't change the fact :)

Interesting, I didn't know about the `include` key, but I agree, for people only building we should remove the `tests/` folder.
So dependencies to `goblin` become slimmer and people wanting to hack on `goblin` will need to clone it from gh anyway.